File: 백준/Bronze/1546. 평균/평균.cs
using System.Linq;

namespace algorithm
{
    internal class Program
    {
        static void Main(string[] args)
         {
            int subject = int.Parse(Console.ReadLine());
            string[] input = Console.ReadLine().Split();
            int[] scores = new int[subject];
            for (int i =0; i < input.Length; i++)
            {
                scores[i] = int.Parse(input[i]);
            }

            float sum = 0;
            int max = scores.Max();
            for (int i = 0; i <subject; i++)
            {
                float newScore = (float)scores[i] / max * 100;
                sum += newScore;
            }
            string result = String.Format("{0:N2}",sum / subject);
            Console.WriteLine(result);
        }
    }
}

File: 백준/Bronze/1546. 평균/평균.cc
#include <iostream>
using namespace std;

int main() {
	int n = 0, input = 0, highScore = 0, sum =0;
	cin >> n;
	for (int i = 0; i < n; i++) {
		cin >> input;
		sum += input;
		if (highScore < input) {
			highScore = input;
		}
	}

	cout << sum / (double)highScore * 100 / n;
}